### PR TITLE
Multi Streamer Support

### PR DESCRIPTION
**Pull request by mcottontensor**
Adding the ability to stream multiple streamers into the one signalling server.

### CODE REVIEW

File: Frontend/library/src/Config/SettingSelect.ts [hunk 1]
```diff
@@ -0,0 +1,139 @@
+// Copyright Epic Games, Inc. All Rights Reserved.
```
Comment (Belchy06) [inline Frontend/library/src/Config/SettingSelect.ts]: Do we still need this file if we've merged to using the SettingOption?

File: SignallingWebServer/cirrus.js [hunk 2]
```diff
@@ -270,387 +270,501 @@ if (config.UseHTTPS) {
...
 	}
-	const playerName = playerId == SFUPlayerId ? "SFU" : `player ${playerId}`;
-	const rawMsg = JSON.stringify(msg);
-	logOutgoing(playerName, msg.type, rawMsg);
-	player.ws.send(rawMsg);
+
+	if (!streamers.has(streamer.id)) {
+		console.error(`Disconnecting streamer ${streamer.id} does not exist.`);
+	} else {
+		sendStreamerDisconnectedToMatchmaker();
```
Comment (lukehb) [inline SignallingWebServer/cirrus.js:391]: Have these changes been tested against the matchmaker?

File: Frontend/library/src/Config/SettingSelect.ts (hunk 1 above)
Comment (mcottontensor) [reply Frontend/library/src/Config/SettingSelect.ts]: No

File: SignallingWebServer/cirrus.js (hunk 2 above)
Comment (mcottontensor) [reply SignallingWebServer/cirrus.js:391]: They have not. I wouldn't even know how to test the MM at this point

Comment (lukehb) [reply SignallingWebServer/cirrus.js:391]: We will focus of automated testing after this release so we don't have to manually do this stuff. But testing MM atm is like:

- Run the .bat in the MatchMaker folder
- Run 1 Cirrus with UseMatchMaker turned on
- Run another Cirrus with UseMatchMaker turned on and different http, ws, and sfu ports, e.g. 81, 7788, 8899
- Run a UE streamer and connect to Cirrus 1, e.g. 8888
- Run another UE streamer and connect to Cirrus 2, e.g. 7788
- Go to browser localhost:80 and localhost:81

Comment (Belchy06) [reply SignallingWebServer/cirrus.js:391]: Just amend last step, you'd wanna go to port 90 as that's the default matchmaker port.

Matchmaker should redirect you to one of the two free sessions.

**Pull request closed without merging**